leasing/management/commands/attach_areas.py

```diff
@@ -22,28 +24,37 @@ class Command(BaseCommand):
     def handle(self, *args, **options):  # noqa: C901 TODO
         from auditlog.registry import auditlog
 
+        logging.basicConfig(level=logging.INFO, format="%(message)s", stream=sys.stdout)
+        log = logging.getLogger(__name__)
+
+        verbosity = options.get("verbosity")
+        if verbosity == 0:
+            log.setLevel(logging.WARNING)
+        elif verbosity == 2:
+            log.setLevel(logging.DEBUG)
+
         # Unregister all models from auditlog when importing
         for model in list(auditlog._registry.keys()):
             auditlog.unregister(model)
 
         leases = Lease.objects.all()
 
+        log.info(f"Processing {leases.count()} objects.")
+
         for lease in leases:
-            self.stdout.write("Lease #{} {}:".format(lease.id, lease.identifier))
+            log.debug("Lease #{} {}:".format(lease.id, lease.identifier))
 
             lease_areas = {
                 la.get_normalized_identifier(): la for la in lease.lease_areas.all()
             }
-            self.stdout.write(
-                " Existing lease areas: {}".format(", ".join(lease_areas.keys()))
-            )
+            log.debug(" Existing lease areas: {}".format(", ".join(lease_areas.keys())))
```

Comment:
The same formatting point applies to all of these messages with `{}` in them, but I won't repeat the comment anymore. :slightly_smiling_face: